---
layout: page
title: About
permalink: /about/
---

I am a group leader at [Institut Pasteur](https://research.pasteur.fr/en/member/sebastian-duchene-garzon/), Paris, France (since October 2023), and I have a split appointment at the [Doherty Institute](https://www.doherty.edu.au/people/dr-sebastian-duchene) in the University of Melbourne. Prior to coming to France I spent most of my time in Australia. I was awarded my PhD at the University of Sydney in 2015, with an award from the Faculty of Science and from the Genetics Society of Australasia. I also hold the title of Accredited Statistician from the Australian Statistical Society. After my PhD I conducted postdoctoral work at the same institution for 18 months. In 2016 I joined the University of Melbourne through an internal fellowship scheme, the McKenzie fellowship, and have been consistently funded by extramural sources. From 2019 to 2022 I was a Discovery Early Career Fellow (DECRA - Australian Research Council). From 2022 I was junior group leader (equivalent to Assistant Professor) funded through a Future Fellowship from the Australian Research Council and a National Health and Medical Research Council Emerging leader Level 2.

I have supervised four postdoctoral researchers through my own funding, three PhD students to completion, four Masters in bioinformatics, and one in honours. My research outputs have resulted in over [10K citations and a H-index of 46](https://scholar.google.com.au/citations?user=K7q8WywAAAAJ&hl=en&oi=ao).

My research has been in the field of microbial computational molecular evolution, with the following major contributions:

**Molecular evolutionary rates in viruses and bacteria**


Two landmark publications (Duchêne et al. 2014; Duchêne et al. 2016) showed the limits of how fast or slow viruses and bacteria can evolve. This work was essential for my subsequent analyses of ancient DNA data of several microbes (Weyrich et al. 2017; Eaton et al. 2023). I have also written reviews and opinion pieces on ancient microbial DNA evolution (Duchêne et al. 2020; Duchene 2023). This work has had profound consequences for our understanding of the evolutionary rate of SARS-CoV-2 (Duchene et al. 2020 Virus Evol), and the mechanism behind the emergence of variants of concern (Tay et al. 2022 Mol Biol Evol). These studies have been covered by various media outlets, including the ABC, Le Figaro (France), El Pais (Spain), The Guardian, and others.

**Techniques to estimate evolutionary timescales**


I have pioneered methods for improving estimates of when pathogens emerge. In particular, I developed two statistical tests that are widely adopted in the field of pathogen molecular genomics and ancient DNA research (Duchene et al. 2015 Mol Biol Evol; Duchene et al. 2020 Mol Biol Evol).

**Methods for estimating epidemiological parameters from molecular sequence data sets**


My team has developed a range of methods for integrating genomic and epidemiological information in sequence data sets (e.g. Featherstone et al. 2020 Mol Biol Evol), some of which I implemented into the widely popular BEAST2.5 software (Bouckaert et al. 2019 Plos Comp Bio).
